src/coral/.npmrc

```diff
@@ -0,0 +1,2 @@
+node-version>=16.0.0<=16.18.0
+engine-strict=true
```

Comment:
Missing newline in the end

Reply:
Ugh, again 🙈  Sorry I didn't catch that. Linting setup should come next I guess 😅